### ISSUE TITLE
"build -prod" of official sample doesn't work (Unexpected token)

### ISSUE THREAD

**Issue by Nasicus** [flauc/angular2-notifications#183]
If you clone this repo, go into the sample directory and execute `npm install` and then `ng build -prod` you get the following error:

```
ERROR in vendor.bd45baaa8347504c51c6.bundle.js from UglifyJs
Unexpected token: name (SimpleNotificationsComponent) [vendor.bd45baaa8347504c51c6.bundle.js:20909,4]
```

Same happens for my "own project".

Version: 0.5.3

I think the problem is, that you use ` "target": "ES6",` in `tsconfig.json` and `uglifyJs` cannot work with this version.

**Comment by psaussure** [comment]
same issue here. Can't use the module for build prod. Exact same message than stated by @Nasicus

**Comment by jotatoledo** [comment]
Same issue. No problem on dev mode

**Comment by Nasicus** [comment]
fixed with version `0.5.6`. Thank you @flauc  & @blackholegalaxy  for the quick fix!

**Comment by sureshokp** [comment]
Error on fetch for node_modules/angular2-notifications/src/simple-notifications.module.js at file:///D:/GIP/IMAGE PILOT/gipfe/node_modules/angular2-notifications/src/simple-notifications.module.js Loading build/tmp/aot/src/app/app.module.ngfactory.js Loading build\boot-aot.js Error: ENOENT: no such file or directory, open 'D:\GIP\IMAGE PILOT\gipfe\node_modules\angular2-notifications\src\simple-notifications.module.js' [10:13:51] 'build.aot' errored after 52 s [10:13:51] Error in plugin 'run-sequence(bundle-aot)' Message: Error on fetch for node_modules/angular2-notifications/src/simple-notifications.module.js at file:///D:/GIP/IMAGE PILOT/gipfe/node_modules/angular2-notifications/src/simple-notifications.module.js Loading build/tmp/aot/src/app/app.module.ngfactory.js Loading build\boot-aot.js ENOENT: no such file or directory, open 'D:\GIP\IMAGE PILOT\gipfe\node_modules\angular2-notifications\src\simple-notifications.module.js' Details: originalErr: Error: ENOENT: no such file or directory, open 'D:\GIP\IMAGE PILOT\gipfe\node_modules\angular2-notifications\src\simple-notifications.module.js' Stack: Error on fetch for node_modules/angular2-notifications/src/simple-notifications.module.js at file:///D:/GIP/IMAGE PILOT/gipfe/node_modules/angular2-notifications/src/simple-notifications.module.js Loading build/tmp/aot/src/app/app.module.ngfactory.js Loading build\boot-aot.js Error: ENOENT: no such file or directory, open 'D:\GIP\IMAGE PILOT\gipfe\node_modules\angular2-notifications\src\simple-notifications.module.js'

build error is came after i update node modules using npm install command then, i try to take build using npm run build.aot command then above errors was came

pls help me to solve this.
